tendrl/node_agent/flows/generate_journal_mapping/__init__.py

```diff
@@ -9,7 +9,8 @@ class GenerateJournalMapping(flows.BaseFlow):
     def run(self):
         # Generate the journal mapping for the nodes
         mapping = utils.generate_journal_mapping(
-            self.parameters['Cluster.node_configuration']
+            self.parameters['Cluster.node_configuration'],
+            integration_id=self.parameters.get("TendrlContext.integration_id")
```

Comment:
generate_journal_mapping does not take more than 1 argument, youll need to fix that